ocs_ci/utility/utils.py

```diff
@@ -1426,7 +1422,56 @@ def get_next_version_available_for_upgrade(current_tag):
             if config.UPGRADE.get("use_rc_build") and "rc" not in tag['name']:
                 continue
             return tag['name']
-    raise TagNotFoundException(f"Couldn't find any tag!")
+    raise TagNotFoundException("Couldn't find any tag!")
+
+
+def get_ocs_olm_operator_tags(limit=100):
+    """
+    Query the OCS OLM Operator repo and retrieve a list of tags.
+
+    Args:
+        limit: the number of tags to limit the request to
+
+    Raises:
+        requests.RequestException: if the response return code is not ok
+
+    Returns:
+        requests.request: response of the request
```

Comment:
Wouldn't be better to return directly the list of tags, instead of the whole response object? (like:
```python
return resp.json()['tags']
```
... with possibility to better handle any other exception there)?

Reply:
I originally didn't want to change any of the logic and just wanted to cut down on duplication but this is a very small change and makes sense. Done!